packages/lit-protocol-cipher/src/lit-protocol-cipher-provider.ts

```
@@ -67,15 +72,33 @@ export default class LitProvider implements CipherProviderTypes.ICipherProvider
+    // Using process.browser instead of typeof window
     if (typeof window !== 'undefined') {
-      return new LitJsSdk.LitNodeClient({
+      this.client = new LitJsSdk.LitNodeClient({
         litNetwork: this.network,
       });
+      await this.client.connect();
     } else {
-      return new LitJsSdk.LitNodeClientNodeJs({
+      // Create dynamic import URL to prevent webpack analysis
+      const moduleUrl = 'node-localstorage';
+      const { LocalStorage } = await (0, eval)(`import('${moduleUrl}')`);
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)